### DISCUSSION TITLE
Idea: Generic thruster set

### DISCUSSION THREAD

**Discussion by Zitchas** [endless-sky/Endless-Sky-Delta#176] [Ideas]
I think the lateral PR makes a logical point to add something else that I've been thinking of: Generic thruster set. Currently, while we try to portray Ionics in a positive light, there's really not much reason to use them. Someone actually buying them is very rare, since if they need to buy them, their ship has plasmas or atomics, and quite a few ships come stock with engines that do come with ionics can't be upgraded to either of the other two options without needing to tweak a bunch of other systems. The fact that they need to make some re-adjustments to make use of those much better engines isn't a problem; the problem is a) there's no "first step" upgrade to get people into the practice of personalizing their ship, and b) that ionics, despite being portrayed as advantageous in some situations, get short-shift. 

This is, fundamentally, a game about ship customization. People feel good when they make a change to their ship that results in it feeling better to use, whatever that use-case happens to be. Having stock ships come pre-fitted at optimum or near-optimum levels where the player really has to work and reconfigure a lot of systems in order to make minor changes is thus problematic. Ideally, the player has a range of early upgrade options that are pure upgrades with little to no reconfiguration required, but correspondingly small changes. Just enough to be notable. So they start off with the stock ship, then upgrade shortly thereafter to ionics to get a little more performance and some better efficiency, and then when they feel the need they start looking at Plasmas and Atomics and figuring out how to reconfigure the ship for larger increases.

So the solution to me is a fourth tier of engines for human space. The generic reaction thrusters. In terms of stats, these are thrusters that have a bit less thrust than ionics, while consuming a bit more energy and producing a bit more heat; and are at about the same size. They do, of course, have an advantage: Cost. Lore being that they are highly reliable and easy to produce, and thus are widely used as "standard thrusters" for new ships. Replacing them is also the first thing that most engineers recommend when a captain is interested in improving the capabilities of their ship.

Part of the intent is to let players actually feel special. To take cars for an example. If every car came from the show floor tweaked to the max, then someone putting some effort into improving their car's performance doesn't stand out. They're just one more car and 99% of people won't be able to tell anything's different. Cars *don't* come tweaked, for the most part. They're built to maximize the profit ratio of the corporation making them, which means standardized construction that shares as many parts as possible with every other vehicle that corporation makes, and generally built for ease of construction, reliability (in the last-long-enough-to-get-out-of-warranty sense, anyway), etc. So when someone installs a V8 sports engine or a turbo on their car that normally just uses a V6, it tends to get noticed by everyone. 

This is quite intentionally going to add another layer of variety to the ships encountered in-game. Fleets made up of stock ships are, by definition, going to be performing a bit worse because they are using the straight-from-the-dealer-generic thrusters; whereas upgraded variants are going to be noticeably better than stock. A player that hasn't changed their engines will just fit right into the general mass of stock ships, while the player that actively tweaks is going to actually feel a step above the general masses of ships. 

This thus also hopefully improves the "Make the player feel good and more powerful while not having them be *special*" sort of thing. Replacing the generics with specific improved engines isn't restricted to the player, after all, but it's something that a lot of captains don't do, so it will tend to make the player stand out a bit. And there's all the variant ships flown by captains that *do* upgrade their ship to keep the pressure on the player.

**Comment by TheGiraffe3** [comment]
Occurs to me that this would also increase the specialty of Marauders, who for the most part use atomic engines - they've gone to great pains outfitting their ship, let alone modifying the more difficult hardware.

**Reply by Zitchas** [reply]
Definitely, yes. They'd feel a lot more special if atomic engines weren't so ubiquitous.

**Comment by Zitchas** [comment]
As an aside, I would also love to get additional thruster serieses in every faction. The only tube they shouldn't have multiple series is if the faction has some major reason against it. Even highly efficient Coalition manufacturing should at least have the thrusters that they developed a free thousand years ago. I'm sure they have come up with a new design in a few thousand years. Same goes for the Wanderers and Korath.
